sketch.js

```diff
@@ -38,199 +40,232 @@ var gameOver = false;
+            damage++;
+        }
     }
-  }
 
-  if(damage >= 450) {
+    if (damage >= 450) {
+        /**
+         * Game Over
+         */
+        invaders = [];
+        image(starsPNG_1080p, displayWidth / 2, displayHeight / 2);
+
+        var damagedShip = new DamagedShip(ship.x, ship.y);
+        damagedShip.show();
+
+        //Stop the draw function
+        noLoop();
+    }
     /**
-     * Game Over
+     * Show the invaders on the canvas
      */
-    invaders = [];
-    image(starsPNG_1080p, displayWidth/2, displayHeight/2);
-
-    var damagedShip = new DamagedShip(ship.x, ship.y);
-    damagedShip.show();
-
-    //Stop the draw function
-    noLoop();
-  }
-
-  /**
-   * Show the invaders on the canvas
-   */
-  var edge = false;
-  for(let i = 0; i < invaders.length; i++) {
-    invaders[i].show();
-    invaders[i].move();
-
-    if(invaders[i].x > width || invaders[i].x < 0) {
-      edge = true;
+    var edge = false;
+    for (let i = 0; i < invaders.length; i++) {
+        invaders[i].show();
+        invaders[i].move();
+
+        if (invaders[i].x > width || invaders[i].x < 0) {
+            edge = true;
+        }
     }
-  }
 
-  for(let b = 0; b < invaderBullets.length; b++) {
-    invaderBullets[b].show();
-    invaderBullets[b].attack();
+    for (let b = 0; b < invaderBullets.length; b++) {
+        invaderBullets[b].show();
+        invaderBullets[b].attack();
 
-    if(invaderBullets[b].hits(ship)) {
-      invaderBullets[b].removeIt();
-      console.log("SHIP WAS HIT BY BULLET!!");
+        if (invaderBullets[b].hits(ship)) {
+            invaderBullets[b].removeIt();
+            console.log("SHIP WAS HIT BY BULLET!!");
+        }
     }
-  }
-
-  if(edge) {
-    for(let i = 0; i < invaders.length; i++) {
-      invaders[i].shiftDown();
-      if(i % 3 == 0) {
-        invaderBullets.push(new Bullet(invaders[i].x, invaders[i].y));
-      }
-    }
-  }
 
+    if (edge) {
+        for (let i = 0; i < invaders.length; i++) {
+            invaders[i].shiftDown();
+            if (i % 3 == 0) {
+                invaderBullets.push(new Bullet(invaders[i].x, invaders[i].y));
+            }
+        }
+    }
 
+    movement();
 }
 
 /**
  * Events
  */
 
 function keyPressed() {
-  if(keyCode == keyCodes.d) {
-    ship.setDirection(1);
-  } else if(keyCode == keyCodes.a) {
-    ship.setDirection(-1);
-  }
-
-  if(keyCode == keyCodes.space) {
-    bullets.push( new Bullet(ship.x, height - 60) );
-    //laserSound.play();
-    laserSound2.play();
-  }
+    // if (keyCode == keyCodes.d) {
+    //     ship.setDirection(1);
+    // } else if (keyCode == keyCodes.a) {
+    //     ship.setDirection(-1);
+    // }
+
+    if (keyCode == keyCodes.space) {
+        dropBoost()
+
+    }
 }
 
 function keyReleased() {
-  if(keyCode != keyCodes.space) {
-    ship.setDirection(0);
-  }
+    if (keyCode != keyCodes.space) {
+        ship.setDirection(0);
+    }
 }
 
+function movement() {
+    if (keyIsDown(65)) {
```

Comment:
Just for for code consistency: it didn't work with keyIsDown(keyCodes.a) and keyIsDown(keykeyCodes.d)?